### PR TITLE
Add io.github.signus_game.signus

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/signus-game/signus/pull/23
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://github.com/signus-game/signus/pull/23

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build io.github.signus_game.signus

File: io.github.signus_game.signus.desktop [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Created with jdDesktopEntryEdit 1.2
+[Desktop Entry]
+Type=Application
+Name=Signus: The Artefact Wars
+GenericName=Turn-based strategy game
+Icon=signus
```
Comment (hfiguiere) [inline io.github.signus_game.signus.desktop:6]: icon name is incorrect.

**Comment by Mailaender** [comment]
bot, build io.github.signus_game.signus

File: io.github.signus_game.signus.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+
+modules:
+  - name: signus
+    buildsystem: autotools
+    build-options:
+      env:
+        CXXFLAGS: "--std=c++03"
```
Comment (hfiguiere) [inline io.github.signus_game.signus.yaml:18]:
```suggestion
      cxxflags: "--std=c++03"
```

File: io.github.signus_game.signus.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    post-install:
+      - install -Dm644 ${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      - install -Dm644 ${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 ${FLATPAK_ID}.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png
+
+  - name: signus-data
+    buildsystem: autotools
+    build-options:
+      env:
+        CXXFLAGS: "--std=c++03"
```
Comment (hfiguiere) [inline io.github.signus_game.signus.yaml:38]:
```suggestion
      cxxflags: "--std=c++03"
```

**Comment by Mailaender** [comment]
bot, build io.github.signus_game.signus

**Comment by ILikePigeons** [comment]
Hi, I don't know if this is the right place to report this bug. So sorry for bothering if it isn't. I have recently noticed that the game won't save due to the fact that it doesn't have access to ~/.signus file directory or it isn't configured to create the config file in the sandbox.

**Pull request closed without merging**